include/revng/PipelineC/PipelineC.h

```diff
@@ -74,6 +74,15 @@ extern "C" {
  * Unless otherwise noted, no pointer arguments can be NULL.
  */
 
+/**
+ * Allow setting a custom traceback function
+ * this will be called just before abort()
+ * in case of an assertion failure
+ * Useful if calling from a non-C language to have
+ * Additional information
```

Comment:
* Weird capitalzation (*U*seful but not `.`, *A*dditional information).
* Wrap at 80 columns.

Reply:
Reworded doc lines accordingly